notebooks/00_HubQuickStart.ipynb

```diff
@@ -1,10 +1,67 @@
 {
```

Comment:
<p>I feel like Spyglass is more than a datapoint pipeline. Specifically:</p><ul><li>it has features related to pipelining that datajoint doesn't have, like cautious delete and merge table class</li><li>it has other features that datajoint doesn't have, like all the functions related to working with NWB (e.g. fetch_nwb etc)</li><li>it integrates other tools for visualization (figurl) and sharing (kachery)</li></ul><p>Because of this I would prefer to call it something like 'framework for reproducible and shareable neuroscientific data analysis' </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass-demo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>